========= test session starts ==========
platform darwin -- Python 3.6.3, pytest-3.3.0, py-1.5.2, pluggy-0.6.0
rootdir: ../python-pair-programming-and-tdd-workshop/test, inifile:
plugins: cov-2.5.1
collected 2 items

test_mymath_more.py .F                               [100%]

=============== FAILURES ===============
_______________ test_add _______________

    def test_add():
        assert mymath.add(2, 2)  == 4
        assert mymath.add(0, 0) == 0
>       assert mymath.add(-1, 1) == 0
E       assert -1 == 0
E        +  where -1 = <function add at 0x102386400>(-1, 1)
E        +    where <function add at 0x102386400> = mymath.add

test_mymath_more.py:10: AssertionError
====== 1 failed, 1 passed in 0.11 seconds ======
